Description: Do not include the ssl23.h file.
 It's not there any more in OpenSSL 1.1.
Forwarded: https://www.stunnel.org/pipermail/stunnel-users/2016-June/005559.html
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2016-06-27

--- a/src/common.h
+++ b/src/common.h
@@ -461,7 +461,6 @@
 
 #include <openssl/lhash.h>
 #include <openssl/ssl.h>
-#include <openssl/ssl23.h>
 #include <openssl/ui.h>
 #include <openssl/err.h>
 #include <openssl/crypto.h> /* for CRYPTO_* and SSLeay_version */
